specificatie/zoek-personen.yaml

```diff
@@ -151,14 +151,13 @@ components:
               description: |
                 Gemeentenaam of een buitenlandse plaats of een plaatsbepaling, die aangeeft waar de persoon is geboren. **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).**
               type: string
-              pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ '\-\']{1,40}(\?+|\*)?$
-              maxLength: 40
+              pattern: ^(\?+|\*)?[a-zA-Z0-9À-ſ \,\.\-\']{1,80}(\?+|\*)?$
```

Comment:
geboorteplaats, als we die al houden, ondersteunt geen wildcards --> [a-zA-Z0-9À-ſ \,\.\-\']{1,80}